lib/ci/package_builder.rb

```diff
@@ -248,7 +248,15 @@ def raise_build_failure
     end
 
     def arch_bin_only?
-      value = ENV.fetch('PANGEA_ARCH_BIN_ONLY', 'true')
+      control = Debian::Control.new(Dir.pwd)
+      control.parse!
+      source_name = control.source.fetch('Source', '')
```

Comment:
wouldn't this be more densely be expressed as 

```ruby
if control.source.fetch('Source').start_with?('qt6-')
  value = ENV.fetch....
```

Reply:
surely does.

Reply:
You still need to construct and parse! a control ;)

Reply:
aaah. that indeed makes sense.